qualtran/bloqs/basic_gates/rotation.py

```diff
@@ -99,6 +99,28 @@ class ZPowGate(_RotationBloq):
     def cirq_gate(self) -> cirq.Gate:
         return cirq.ZPowGate(exponent=self.exponent, global_shift=self.global_shift)
 
+    def __pow__(self, power):
+        g = self.cirq_gate**power
+        return ZPowGate(g.exponent, g.global_shift, self.eps)
+
+
+@frozen
+class CZPowGate(CirqGateAsBloqBase):
+    exponent: float = 1.0
+    global_shift: float = 0.0
+    eps: float = 1e-11
+
+    @cached_property
+    def cirq_gate(self) -> cirq.Gate:
+        return cirq.CZPowGate(exponent=self.exponent, global_shift=self.global_shift)
+
+    def _t_complexity_(self) -> 'TComplexity':
```

Comment:
Should this derive from _RotatationBloq? Otherwise eps is doing nothing here right?

Reply:
I wasn't sure if the T-complexity formula in `_RotationBloq` holds for two qubit rotations as well (instead of just single qubit rotations). Do you know if that is the case? I'd be happy to change it to `_RotatationBloq` if that's true or I can add a TODO to figure it out later.

Reply:
I don't know, TODO SGTM